lib/Newsletter/Renderer/Columns/Renderer.php

```diff
@@ -65,7 +67,7 @@ private function renderMultipleColumns($content_block, $columns_data) {
   private function getMultipleColumnsContainerStart($class, $styles, $image) {
     return '
       <tr>
-        <td class="mailpoet_content-' . $class . '" align="left" style="border-collapse:collapse;' . $this->getBackgroundCss($styles, $image) . '">
+        <td class="mailpoet_content-' . $class . '" align="left" style="border-collapse:collapse;' . EHelper::escapeHtmlStyleAttr($this->getBackgroundCss($styles, $image)) . '" ' . $this->getBgColorAttribute($styles, $image) .'>
```

Comment:
One is escaped inside the method, the other one outside of it. Could this be unified?